server/repository/src/migrations/v2_04_00/indicator_line_column_create_tables.rs

```diff
@@ -0,0 +1,56 @@
+use crate::migrations::*;
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "indicator_column_create_table"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            sql!(
+                connection,
+                r#"
+                CREATE TYPE indicator_value_type AS ENUM (
+                    'STRING',
+                    'NUMBER'
+                );
+                "#
+            )?
+        }
+
+        let value_type = if cfg!(feature = "postgres") {
+            "indicator_value_type"
+        } else {
+            "TEXT"
+        };
+
+        sql!(
+            connection,
+            r#"
+            CREATE TABLE indicator_column (
+                id TEXT PRIMARY KEY NOT NULL,
+                program_indicator_id TEXT NOT NULL REFERENCES program_indicator(id),
+                column_number BIGINT NOT NULL,
```

Comment:
I'm curious do we need big ints for these values? Is this something to do with communicating with 4d data types?

Reply:
Strictly, probably not. it could probably be i32, if not u16 haha. But also I get annoyed with coercing different int types in rust so just doing i64 as default is pretty safe hahaha.

Truly in 4D land it is i32... hmm.

SQLite sorta dynamically adjusts the storage type for integer. 0-255 will just use 1 byte. several quintillion 8 bytes. Postgres is more strict, though. 

FINE 😆.

Reply:
quick re-review pls: https://github.com/msupply-foundation/open-msupply/pull/5295/commits/265c3972e223cabe8b75ca49031dd886d1369ff0